bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml

```diff
@@ -15,7 +15,7 @@
 			<description>Duration of transition of events such as on/off, change of brightness and change of color, in
 				milliseconds.</description>
 			<unitLabel>milliseconds</unitLabel>
-			<default>500</default>
+			<default>5000</default>
```

Comment:
Why this change?

Reply:
There is an issue that device stays offline with default duration 500. I experienced it myself and was able to resolve by increasing duration to 5000.
Also same issue happens to others: https://community.openhab.org/t/yeelight-color-led-bulb-offline-using-yeelight-binding/96583/4

Reply:
So this is not transition time, i.e. the time it takes to from light being on to light being fully off, for example? In that case 5 seconds seems awfully slow as default value, but I may of course have read the description and documentation wrong. Transition time put into a request should not cause those problems mentioned?

Please note that default value is also here:
https://github.com/openhab/openhab-addons/blob/b395d0e2271077409f15c9f7ac17ba3814d064f9/bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightHandlerBase.java#L351-L353

Reply:
Yeah, this value also affects how much is a wait time between sending every new dice command, so setting it to 5s makes device less responsive. There is an option to keep default as 500 but update comment so users will see that this value affects device online status:

https://github.com/openhab/openhab-addons/blob/b395d0e2271077409f15c9f7ac17ba3814d064f9/bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/i18n/yeelight.properties#L30

`Duration of transition of events such as on/off, change of brightness and change of color, in milliseconds. Try to increase if device is going offline.`

Reply:
In context of this PR, in my opinion, we should not change this default.

Would it be possible to explain why transition time and offline problems are correlated, and suggest increasing the value as a work-around? This would be a nice improvement for the README. I'm not sure if adding this to the parameter description would give enough information, and might just raise more questions? It would be okay to improve the documentation in this PR, but you could also create a new dedicated PR to not block this one as it's not related.

Reply:
ok sounds good

Reply:
@jlaur I opened an issue for device going offline https://github.com/openhab/openhab-addons/issues/14635